docs/competition/teams/photo-studio.md

```diff
@@ -0,0 +1,21 @@
+# Photo studio manager
+
+The role of the photo studio manager is to take nice photos of teams with their robots.
+
+You should have:
+- A camera on a tripod.
+- A notebook to annotate each photograph.
+
+By far the most important part of this role is to make sure the photos are in focus and correctly framed.
+
+To do this role please make sure you take note of which photos go to which team! Please write down the ID of the photos for each team (both the ID for the first and last).
```

Comment:
I have found it helpful to take a photo of the TLA at the start and end of each teams photos

Reply:
Fixed in cf7321e